www/app/blog/2024-11-19-distributed-ddl.mdx

```diff
@@ -142,7 +142,7 @@ The transaction coordinator then sends the `PREPARE TRANSACTION` command to all
 
 According to the [Postgres documentation](https://www.postgresql.org/docs/current/sql-prepare-transaction.html), the `PREPARE TRANSACTION` behaves as follows:
 
-> `PREPARE TRANSACTION` prepares the current transaction for two-phase commit. After this command, the transaction is no longer associated with the current session; instead, its state is fully stored on disk, and there is a very high probability that it can be committed successfully, even if a database crash occurs before the commit is requested.  
+> `PREPARE TRANSACTION` prepares the current transaction for two-phase commit. After this command, the transaction is no longer associated with the current session; instead, its state is fully stored on disk, and there is a very high probability that it can be committed successfully, even if a database crash occurs before the commit is requested.
```

Comment:
Sorry my editor stripped trailing whitespace, I'm happy to roll this back if desired

Reply:
Thank you! I'm running a build now to see if the whitespace affects formatting.

Reply:
Apparently markdown requires these 2 trailing spaces in order to format the next line as a newline. Mind bringing them back?

Reply:
Not at all! Give me 5m

Reply:
Done